contracts/ETO/ETOCommitment.sol

```diff
@@ -531,6 +533,15 @@ contract ETOCommitment is
         usedLockedAccount = ticket.usedLockedAccount;
     }
 
+    // recycle all payment tokens held on this contract as a result of NEU proceeds
+    function recycle()
+        public
+        onlyState(ETOState.Payout) {
+        IFeeDisbursal disbursal = IFeeDisbursal(UNIVERSE.feeDisbursal());
+        disbursal.reject(ETHER_TOKEN, NEUMARK, 256**2-1);
+        disbursal.reject(EURO_TOKEN, NEUMARK, 256**2-1);
```

Comment:
It would be nicer to iterate all known paymenttokens, but there is no such function on the universe is there? Future payment tokens then might be lost.